ops.h

```diff
@@ -277,69 +333,105 @@ static HWY_NOINLINE HWY_MAYBE_UNUSED void GeluMulToBF16(
   }
 }
 
+template <class D, HWY_IF_F32_D(D)>
+static HWY_INLINE hn::Vec<D> Sigmoid(D d, hn::Vec<D> v) {
+  using VF = hn::Vec<D>;
+  // Chebyshev polynomial coefficients for rational approximation
+  const VF c0 = Set(d, 0.00949107017368078f);
```

Comment:
Let's add hn:: to all ops, otherwise it will break on SVE (and some people are indeed targeting that).